### DISCUSSION TITLE
Diagnostics

### DISCUSSION THREAD

**Discussion by rasmusludeking** [tiagovla/tokyodark.nvim#41] [General]
This might be a stupid question. But I've followed the very short readme to its fullest. My diagnostics window looks nothing like the presented one. What might i be doing wrong?

**Comment by shreyas-a-s** [comment]
By 'nothing like presented one', do you mean the colors are different? Or you mean the diagnostics menu is entirely different?

**Comment by rasmusludeking** [comment]
![image](https://github.com/tiagovla/tokyodark.nvim/assets/18356378/c2521382-e184-4db4-ad2d-70d6f3a7d7b2)
Its very hard to see, due to the fact that the background colors are the same, but in the picture; vim.diagnostic.open_float is called and opened. The theme is very nicely done, and I would be very sad not to have this feature work properly.

**Comment by rasmusludeking** [comment]
UPDATE
![image](https://github.com/tiagovla/tokyodark.nvim/assets/18356378/9b6ec529-e5a1-4fb6-8da4-2f86ad60f4cd)
I tried with Lspsaga, but issue still persists.

**Comment by shreyas-a-s** [comment]
Looking at this I suspect at some point, @tiagovla might have changed the colors a bit and the README image just stayed the same.

**Reply by rasmusludeking** [reply]
oof, that's a rip. That diag float is the main selling point. Is there any way I might be able to reproduce it?

**Reply by shreyas-a-s** [reply]
I think Tiago's dotfiles might lead you somewhere. Because I think the way diagnostics looks in that image could have something to do with how he configured it. You could check it out.

[Link](https://github.com/tiagovla/.dotfiles)

**Reply by rasmusludeking** [reply]
thank you very much

**Reply by shreyas-a-s** [reply]
You're welcome, my friend.

**Comment by tiagovla** [comment]
The popup one? That was the old lspsaga.nvim plugin. I don't use it anymore, did it change?

**Reply by rasmusludeking** [reply]
Sorry, was looking through your .dotfiles trying to figure out how you done it. Yeah, the most flashy part about this colorscheme is definitely the diagnostics float. But you maybe right, if I have missed nothing and by installing the colorscheme and lspsaga - it should visualize as in the readMe image, an updated lspsaga must be the culprit - sadly. EDIT: Nn my previous comments ive included the result of installing tokyodark alongside lspsaga.

**Reply by tiagovla** [reply]
Unfortunately, I can't check this at the moment as I'm working on another project. I will investigate why the lspsaga ui is not loading. I think this might not be related to the colorscheme.

**Reply by tiagovla** [reply]
It seems that lspsaga changed. I get this as diagnostics:
![image](https://github.com/tiagovla/tokyodark.nvim/assets/30515389/74f4a3ac-4de6-4bc9-abc9-7b6fe1552df1)

It looks close to this gif:
https://github.com/nvimdev/lspsaga.nvim/issues/1052#issuecomment-1531198523

It might be possible to change the layout, it's in lspsaga config and colors.
Could you test it in another colorscheme and see if you get the same results?

**Comment by rasmusludeking** [comment]
![image](https://github.com/tiagovla/tokyodark.nvim/assets/18356378/916626c9-1654-44e8-9e75-dc5da6a1626a)
With tokyo, excuse the colors, I was messing around with highlights.

![image](https://github.com/tiagovla/tokyodark.nvim/assets/18356378/d4e99ed8-7042-48ac-bc44-60ce879381cb)
Without tokyo

![image](https://github.com/tiagovla/tokyodark.nvim/assets/18356378/55068b9d-c7df-4295-9553-44feacc2a3cf)
With catpuccine
